app/controllers/api/v1/people_controller.rb

```diff
@@ -25,13 +25,13 @@ def update
 
       def image
         person = Person.find(params[:person_id])
-        image_data = NomisClient::Image::get(person.latest_profile.latest_nomis_booking_id)
 
-        if image_data
-          send_data image_data, type: 'image/jpg', disposition: 'inline'
-        else
-          render status: :not_found
+        if !person.picture.attached?
+          image_blob = NomisClient::Image::get(person)
+          person.attach_picture(image_blob)
         end
+
+        render json: { url: url_for(person.picture) }
```

Comment:
don't think this is right. I think JSON API spec requires a data: element with an array in it

Reply:
Fixed, this is now using a serializer.